models/follows.js

```diff
@@ -0,0 +1,10 @@
+module.exports = (sequelize, DataTypes) => {
+  const follows = sequelize.define('follows', {
+    followerId: DataTypes.INTEGER,
+    followingId: DataTypes.INTEGER
+  }, {});
+  follows.associate = () => {
```

Comment:
I think you should take out this association function since you are not using it.